### PR TITLE
Pass selected workflow from AS controller to ASCreateService

### LINKED ISSUE #3161: Creating an admin set is broken in 7.3.0
### Descriptive summary

When an admin user fills out the form to create a new admin set and tries to save, the following error occurs:

`ArgumentError in Sufia::Admin::AdminSetsController#create`
`wrong number of arguments (given 2, expected 3)`

The problem is the initializer in AdminSetCreateService is defined with `def initialize(admin_set, creating_user, workflow_name)` while the create_admin_set method in AdminSetsController is passing `AdminSetCreateService.new(@admin_set, current_user).create` with no workflow name.

See https://github.com/projecthydra/sufia/blob/v7.3.0/app/controllers/sufia/admin/admin_sets_controller.rb#L98
and
https://github.com/projecthydra/sufia/blob/v7.3.0/app/services/sufia/admin_set_create_service.rb#L19

Passing the contents of `params[:admin_set][:workflow_name]` as a third param does fix it.

### Steps to reproduce the behavior

1. Generate a test app, an admin user, and create a default admin set
2. Go to /admin/admin_sets/new, fill out the form, and try to save 

Looks like the bug was introduced in https://github.com/projecthydra/sufia/commit/d49df2fd4577c674928069696d758b68b393d72a

### PR DESCRIPTION
**Pull request by mjgiarlo**
* Allow AS controller to pass workflow_name param to service (thanks, @hortongn!)
 * Default ASCreateService to the default workflow if `nil` is passed
 * Add a test for the rake task

Fixes #3161

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Took another pass at this, @jcoyne, and you may want to re-review.

File: app/controllers/sufia/admin/admin_sets_controller.rb [hunk 1]
```diff
@@ -95,7 +95,7 @@ def list_search_builder
       end
 
       def create_admin_set
-        AdminSetCreateService.new(@admin_set, current_user).create
+        AdminSetCreateService.new(@admin_set, current_user, selected_workflow).create
```
Comment (jcoyne) [inline app/controllers/sufia/admin/admin_sets_controller.rb]: Where is selected_workflow defined?

Comment (mjgiarlo) [inline app/controllers/sufia/admin/admin_sets_controller.rb]: Emacs betrayed me. Changes pushed.

Comment (jcoyne) [inline app/controllers/sufia/admin/admin_sets_controller.rb]: That's the nature of Emacs, https://chocolatey.org/content/packageimages/vim.8.0.568.svg would never do that.

File: spec/tasks/rake_spec.rb [hunk 2]
```diff
@@ -23,5 +26,15 @@
       expect(IO.read("abc123.txt")).to include(user1.email, user2.email)
       File.delete("abc123.txt")
     end
+
+    describe 'sufia:default_admin_set:create' do
```
Comment (jcoyne) [inline spec/tasks/rake_spec.rb]: This test does not pertain to the commit message.  Make this a separate commit?

Comment (mjgiarlo) [inline spec/tasks/rake_spec.rb]: Done.

**Pull request merged**